### ISSUE TITLE
Environment

### ISSUE THREAD

**Issue by javierpastorfernandez** [gymoon10/Instance-Segmentation-with-Discriminative-Loss#1]
Hi, great work!
¿Is there anywhere I could look up the environment requirements or dependencies to run the code?

**Comment by gymoon10** [comment]
Thank you for showing interest in my repository. I hope it was helpful, even if just a little.

I didn't create a separate requirements.txt file, so it's hard to know the exact dependencies. And I actually remember that the original code(https://github.com/Wizaron/instance-segmentation-pytorch) did not specify any dependencies either. However, when I was working on the project in March 2022, I didn't specify any particular versions when installing the packages except torch. The torch version was
conda install pytorch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0 cudatoolkit=11.3 -c pytorch (CUDA 11.3)

I will check if the virtual environment folder used at that time still exists, and if it does, I will organize it and let you know as soon as possible.

**Comment by gymoon10** [comment]
@javierpastorfernandez 

I couldn't retrieve the virtual environments I used previously, so I confirmed that the code works in my current environment and uploaded the **requirements.txt** file. However, in the current package environment I'm using, the **.next()** method of the torch DataLoader is deprecated, so I had to modify the code model.py. **Please use model_update.py & train3_update.py instead of model.py & train3.py.** However, I cannot guarantee that the modifications are entirely correct. I plan to review it further in the meantime.

**If you prefer to use the original code, you can install a version of torch where the .next() method (line 259 of model.py) is not deprecated, and continue using model.py as is.**

**Comment by javierpastorfernandez** [comment]
Thank you so much!

**Comment by gymoon10** [comment]
Please give it a try and leave an issue again if you have any problems. Thank you.
